### PR TITLE
Manually document System.IO.File, FileInfo, FileSystemWatcher

### PR DESCRIPTION
**Pull request by carlossanlop**
@JeremyKuhne and I are owners.
@mairaw and @rpetrusha for language review.

### CODE REVIEW

File: xml/System.IO/FileSystemWatcher.xml [hunk 1]
```diff
@@ -643,7 +643,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Releases the unmanaged resources used by the <see cref="T:System.IO.FileSystemWatcher" />.</summary>
```
Comment (rpetrusha) [inline xml/System.IO/FileSystemWatcher.xml]: I have no idea why, but the parameterless Dispose method does not appear in the build, while the Dispose(Boolean) method does. @dend and @joelmartinez, can you take a look?

File: xml/System.IO/FileSystemWatcher.xml [hunk 2]
```diff
@@ -1027,8 +1027,18 @@
...
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Releases the resources held by the current instance.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+> [!NOTE]
+
```
Comment (rpetrusha) [inline xml/System.IO/FileSystemWatcher.xml]:
```suggestion
```

File: xml/System.IO/FileSystemWatcher.xml [hunk 3]
```diff
@@ -1027,8 +1027,18 @@
...
+Application code does not call this method; it is automatically invoked during garbage collection, unless finalization by the garbage collector has been disabled. For more information, see <xref:System.GC.SuppressFinalize%2A?displayProperty=nameWithType> and <xref:System.Object.Finalize>. This method overrides <xref:System.Object.Finalize>.
```
Comment (rpetrusha) [inline xml/System.IO/FileSystemWatcher.xml]:
```suggestion
> Application code does not call this method; it is automatically invoked during garbage collection, unless finalization by the garbage collector has been disabled. For more information, see <xref:System.GC.SuppressFinalize%2A?displayProperty=nameWithType> and <xref:System.Object.Finalize>. This method overrides <xref:System.Object.Finalize>.
```

File: xml/System.IO/FileInfo.xml [hunk 4]
```diff
@@ -1384,10 +1386,42 @@ C:\Users\userName\AppData\Local\Temp\tmp70CB.tmp was successfully deleted.
...
+## Examples
+
+The following example demonstrates moving a file to a different location and renaming the file.
+
+[!code-csharp[IO.FileInfo.MoveTo#1](~/samples/snippets/csharp/VS_Snippets_Misc/IO.FileInfo.MoveTo/CS/Program.cs#1)]
+[!code-vb[IO.FileInfo.MoveTo#1](~/samples/snippets/visualbasic/VS_Snippets_Misc/IO.FileInfo.MoveTo/VB/Program.vb#1)]
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.IO.IOException">An I/O error occurs, such as the destination device is not ready.</exception>
```
Comment (rpetrusha) [inline xml/System.IO/FileInfo.xml]:
```suggestion
        <exception cref="T:System.IO.IOException">An I/O error occurred, such as the destination device is not ready.</exception>
```

File: xml/System.IO/File.xml [hunk 5]
```diff
@@ -477,7 +477,7 @@
         <param name="path">The file to append the specified string to.</param>
         <param name="contents">The string to append to the file.</param>
         <param name="encoding">The character encoding to use.</param>
-        <summary>Appends the specified string to the file, creating the file if it does not already exist.</summary>
+        <summary>Opens a file, appends the specified string to the file, creating the file if it does not already exist.</summary>
```
[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.IO/File.xml]: No problem. I wanted all overloads to look similar.

File: xml/System.IO/FileSystemWatcher.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.IO/FileSystemWatcher.xml]: @dend @joelmartinez any chance you could help us? This is the only thing blocking us.

Comment (joelmartinez) [reply xml/System.IO/FileSystemWatcher.xml]: Hmm ... this API doesn't seem to be added to any of the FrameworkIndex files (a file managed by mdoc). A visual inspection of the [`Dispose()` member](https://github.com/dotnet/dotnet-api-docs/blob/f121abe96f22ee4c71c4558bae03bfbee81debfc/xml/System.IO/FileSystemWatcher.xml#L619) suggests that it should be showing up. 

@mimisasouvanh I think this probably needs to be a devops bug to be looked at

@rpetrusha @carlossanlop it looks like this issue doesn't have anything to do with this particular PR, so it shouldn't hold up the merge. We'll have to look into the issue and resolve it in a future release of mdoc. Good find 👏

Comment (carlossanlop) [reply xml/System.IO/FileSystemWatcher.xml]: Thank you!

**Comment by carlossanlop** [comment]
@rpetrusha @mairaw the build passed and the last comment has been addressed (not blocking). I think we can merge.

File: xml/System.IO/FileSystemWatcher.xml (hunk 1 above)
Comment (mimisasouvanh) [reply xml/System.IO/FileSystemWatcher.xml]: @joelmartinez @carlossanlop I know the PR was merged, but was there a devops bug created for the missing Dispose() member issue? If not, I can create it.

Comment (carlossanlop) [reply xml/System.IO/FileSystemWatcher.xml]: I did not create an issue. Maybe @joelmartinez did?

Comment (joelmartinez) [reply xml/System.IO/FileSystemWatcher.xml]: @mimisasouvanh I did not create a devops issue

**Pull request merged**